scripts/fast-reboot

```diff
@@ -532,6 +565,9 @@ if [[ -x /usr/bin/watchdog ]]; then
     /usr/bin/watchdog -e
 fi
 
+# Reset swss restart time
+sed -i -e "s/\<RestartSec=300\>/RestartSec=30/" /etc/systemd/system/swss.service || /bin/true
```

Comment:
Please make it a function since it was called twice.

Reply:
Removed the other call from the trap handler, so still keeping this here.